bitnami/mariadb/Chart.yaml

```diff
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 11.0.14
+version: 11.1.14
```

Comment:
If you decide to upgrade the minor version it would be `11.1.0`

Reply:
You're right, thanks! I fixed it